docs/src/main/asciidoc/reactive-routes.adoc

```diff
@@ -52,27 +52,26 @@ import io.vertx.ext.web.RoutingContext;
 
 import javax.enterprise.context.ApplicationScoped;
 
-@ApplicationScoped <1>
+@ApplicationScoped // <1>
```

Comment:
The number seems be displayed well, why did you add "//"?

Reply:
Hum, it didn't when I generated locally (the copy and paste was messed up). 
I will check on the website.